modules/op-understanding-credential-selection.adoc

```diff
@@ -0,0 +1,42 @@
+// This module is included in the following assembly:
+//
+// *openshift-docs/cicd/pipelines/authenticating-pipelines-using-git-secret.adoc
+
+[id="op-understanding-credential-selection_{context}"]
+= Understanding credential selection
+
+A pipeline run or task run may require multiple authentications for accessing different Git repositories. You must annotate each secret to specify the domains for which {pipelines-shortname} can use the credential contained in a  secret.
+
+A credential annotation key for Git secrets must begin with `tekton.dev/git-`, and its value is the URL of the host for which you want {pipelines-shortname} to use that credential.
+
+For example, {pipelines-shortname} uses a `basic-auth` secret (username and password) to access repositories at `github.com` and `gitlab.com`.
+
+.Example: Multiple credentials for basic authentication
+[source,yaml,subs="attributes+"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  annotations:
+    tekton.dev/git-0: github.com
+    tekton.dev/git-1: gitlab.com
+type: kubernetes.io/basic-auth
+stringData:
+  username: <cleartext username>
+  password: <cleartext password or personal access token>
+----
+
+You can also use a `ssh-auth` secret (private key) to access a Git repository.
```

Comment:
```suggestion
You can also use an `ssh-auth` secret (private key) to access a Git repository.
```
Yes, it's strange, but choosing "a" or "an" depends on the _sound_ of the following consonant. Here, because SSH is pronounced "ess-ess-ayche," we use "an."